File: 677/677A.py
n,h = [int(i) for i in input().split()]
a = [int(i) for i in input().split()]
max_width = 0
for i in range(len(a)):
    if a[i] > h :
        max_width += 1
max_width = max_width + n
print(max_width)

File: 677/677A.cpp
#include<iostream>
using namespace std;
int main(){
    int n,h,a;
    cin >> n >> h;
    int max_width = n;
    for (int i=0;i<n;i++){
        cin >> a;
        if (a>h){
            max_width = max_width + 1;
        }
    }
    cout << max_width;
}